tools/cldr-apps/js/src/views/AnnouncePost.vue

```diff
@@ -50,6 +48,11 @@ export default {
         .replaceAll("\n", "\n<br>\n")
         .replaceAll(/(http\S+)/g, "<a href='$1'>$1</a>");
     },
+    localeText() {
+      if (!this.announcement.locs) return "All locales";
+      if (this.announcement.locs === "!") return "DDL Locales";
```

Comment:
Is there some inconsistency between "DDL Locales" and "Non-TC locales"?

Also some duplication of code between describeLocs() and localeText()?

Reply:
My thought was that it should be called "DDL Locales" to users, that will make more sense than "non-TC". However, for the implementation, it's clearer to say "non-TC".

Reply:
Hmmm... If that's by design, ideally it could be documented with comments in the source code, and encapsulated in one place (or one place on the back end and one place on the front end). Is it followed consistently? There's a displayed message, "(empty for all locales, '!' for non-TC)", which seems to be an exception... I also suspect that if the distinction between DDL and non-TC is important from a technical perspective, sooner or later it will matter to the users as well and they should see accurate/consistent terminology even if it's less intuitive...